django/contrib/admin/static/admin/js/inlines.js

```diff
@@ -272,4 +272,15 @@
 
         return $rows;
     };
+
+    $(document).ready(function() {
+        $(".js-inline_admin_formset").each(function() {
+            var inline_options = JSON.parse(JSON.parse('"' + $(this).text() + '"'));
```

Comment:
do you think it would be better to add https://github.com/fusionbox/django-argonauts/blob/master/argonauts/templatetags/argonauts.py as a private admin template tag?